src/lhttpc.app.src

```diff
@@ -30,11 +30,13 @@
 %%% @end
 {application, lhttpc,
     [{description, "Lightweight HTTP Client"},
-        {vsn, "1.3.0"},
+        {vsn, git},
         {modules, []},
         {registered, [lhttpc_manager]},
         {applications, [kernel, stdlib, ssl, crypto]},
   {mod, {lhttpc, nil}},
-  {env, [{connection_timeout, 300000}, {pool_size, 50}]}
+  %% make default pool ~ all possible ports for a single IP interface. e.g. Usual 1025-65535
+  %% define your pool or other size if you want to limit it or have more IP interfaces
+  {env, [{connection_timeout, 300000}, {pool_size, 64000}]}
```

Comment:
I agree with raising the default limit, I am not sure 3 orders of magnitude increase is needed to ameliorate the problem, and I wonder if it's not likely to result in other unexpected problems (like OOM due to thousands of in-flight requests). Pool size is a back pressure mechanism, and I suspect that there are clients that rely on this subconsciously for whom this could cause a significant unexpected problem. is there a problem with just raising to, say, 500 or 1000?